### ISSUE TITLE
There is a white line around the bottom and right sides of the Save First button when quitting the Character Editor.

### ISSUE THREAD

**Issue by daerogami** [calref/cboe#67]
This is an item logged from the To-Do. If you are familiar with or can recreate this bug, please provide details here.

**Comment by NQNStudios** [comment]
This is referring to the intentional highlighting of the default button. 

<img width="461" alt="Image" src="https://github.com/user-attachments/assets/3e08ac7f-7ad0-4a61-9794-3f6da50c3fa2" />

On some buttons, one or more sides of the white border get covered up so it looks buggy.

**Comment by NQNStudios** [comment]
Ah -- the inset visual effect just looks worse on dark backgrounds. I wonder how it can be made better?

**Comment by CelticMinstrel** [comment]
I don't think that would help in the current implementation. It's just an outline, not a filled rect or anything.
